### DISCUSSION TITLE
Feature: Add related posts to footer of post

### DISCUSSION THREAD

**Discussion by gummigroda** [lxndrblz/anatole#283] [Ideas]
Hi,

It would be nice to have the functionality of [Related posts](https://gohugo.io/content-management/related/) as an optional footer on each post.
This makes the flow of the site better for the users interested in similar posts with a more in depth link.

**Comment by lxndrblz** [comment]
@gummigroda This feature has been implemented in version [v1.9.0](https://github.com/lxndrblz/anatole/releases/tag/v1.9.0).
